packages/generator-one-app-module/.eslintignore

```diff
@@ -0,0 +1,2 @@
+test-results
```

Comment:
This file can be removed bc this stuff is now added to the top level `.eslintignore`.